src/nvim/CMakeLists.txt

```diff
@@ -326,7 +326,9 @@ add_custom_command(
 
 add_custom_command(
   OUTPUT ${VIM_MODULE_FILE}
-  COMMAND ${LUA_PRG} ${CHAR_BLOB_GENERATOR} ${VIM_MODULE_FILE}
+  COMMAND ${CMAKE_COMMAND} -E env
+      "LUAC_PRG=${LUAC_PRG}"
+      ${LUA_PRG} ${CHAR_BLOB_GENERATOR} ${VIM_MODULE_FILE}
```

Comment:
Windows runners don't seem to like this and complain about `cmake -E env: No command given`. Anyone with more Cmake knowledge know how to address this? (poke @jamessan).

Reply:
I think we need someone with a Windows dev environment setup to test this and see what's going wrong.  @erw7?

Reply:
I think we need to do some escaping. I have tried several things, but they do not work. On Windows, how about using `loadstring` and `string.dump` as follows? This method is about `3k` larger (`vim.lua`: 20k, pre-compile: 8k, this method: 11k), but the load time doesn't seem to change much.

```lua
local compiled = string.dump(assert(loadstring(sourc:read('*a'), source_name)))
```

Reply:
It looks like https://github.com/neovim/neovim/pull/16631/commits/a1492c830c69950a4255df9614dac81c7540b14a solved the first problem, but now there are other errors :)